lib/brainstem/query_strategies/filter_or_search.rb

```diff
@@ -49,19 +49,20 @@ def run_search(scope, includes, sort_name, direction)
           order: { sort_order: sort_name, direction: direction },
         )
 
+        search_options.reverse_merge!(@options[:primary_presenter].extract_filters(@options[:params], @options))
```

Comment:
This would probably be a good situation to create a new `query_strategy`, because this possibly changes how your presenter and search block need to interact with Brainstem. It would probably be better to keep this working as is for legacy reasons.